light-client/verifier/Cargo.toml

```
@@ -11,6 +11,7 @@ base64 = "0.21"
 displaydoc = "0.2"
 prost = { version = "0.11", default-features = false, features = ["prost-derive"] }
 serde = { version = "1.0", default-features = false, features = ["alloc", "derive"] }
+serde_with = { version = "3.0", features = ["hex"] }
```

:shrug: I'm not sure how much it matters, and I know we need to revisit our dependabot and dependencies stance, but if this uses version 2.3 it will minimize the Cargo.lock changes
```suggestion
serde_with = { version = "2.3", features = ["hex"] }
```